extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -90,14 +92,15 @@ function VideoWrapperWithRef(
     src,
     sources,
     poster,
-    onLoad,
+    onReadyState,
     ...rest
   },
   ref
 ) {
   useResourcesNotify();
```

Comment:
Still necessary, but strictly for integration with non-V1 components. Hopefully gone soon.

Reply:
When  can it be removed?

Reply:
My expectation that it's actually now only used for the accordion when the element's size can change significantly from render to render. I can't think of any other dependency we have now. And I'd say it'd have to stay until either we move every component to V1 or we try to force through the `intersect-resources` or some alternative to it.

Reply:
Thinking out loud: would it make sense to hook up a ResOb to `mutateElement` ? Then only `<amp-accordion>` would be calling mutateElement, and only when size changes, instead of each component on each render.

Reply:
Yeah. That could work, but I need to do a better analysis what we've being using it for. Maybe @jridgewell remembers better. It was always meant to be a temporary thing. Definitely outside of this PR though.